mailpoet/tasks/code_sniffer/composer.json

```diff
@@ -18,7 +18,7 @@
   ],
   "config": {
     "platform": {
-      "php": "7.1.8"
+      "php": "8.1"
```

Comment:
Do you think we could go with a minimum version requirement here (like `>= 8.0` or something) or does it need to be exactly `8.1`?

Reply:
`slevomat/coding-standard` requires > 8.0 so I had to bump this.

`>=` doesn't appear to work here, and the previous version was specifically defined. So I think its ok to define 8.1. I know 8.1 works anyway!